tests/runtime/http_server/test_http_server.py

```diff
@@ -544,6 +545,78 @@ def test_health_check_ok(runtime_http_server):
         assert response.text == "OK"
 
 
+def test_runtime_info_ok(runtime_http_server):
+    """Make sure the runtime info returns version data"""
+    with TestClient(runtime_http_server.app) as client:
+        response = client.get(http_server.RUNTIME_INFO_ENDPOINT)
+        assert response.status_code == 200
+
+        json_response = json.loads(response.content.decode(response.default_encoding))
+        assert "caikit" in json_response
+        assert "runtime_image" not in json_response
+        # dependent libraries not added if sys_modules not set to true
+        assert "py_to_proto" not in json_response
+
+
+def test_runtime_info_ok_response_all_sys_modules(runtime_http_server):
+    with temp_config(
+        {
+            "runtime": {
+                "versioning": {
+                    "sys_modules": True,
+                    "runtime_image": "1.2.3",
+                    "foo": "bar",
+                }
+            },
+        },
+        "merge",
+    ):
+        with TestClient(runtime_http_server.app) as client:
+            response = client.get(http_server.RUNTIME_INFO_ENDPOINT)
+            assert response.status_code == 200
+
+            json_response = json.loads(
+                response.content.decode(response.default_encoding)
+            )
+            assert json_response["runtime_image"] == "1.2.3"
+            assert "caikit" in json_response
+            # dependent libraries versions added
+            assert "alog" in json_response
+            assert "py_to_proto" in json_response
+            # additional config values ignored
+            assert "foo" not in json_response
+
+@patch.dict(
```

Comment:
I also think we don't need specific tests for env var overrides on this once it's part of the standard `caikit.config` since that's standard and assumed to be tested in `aconfig` iteself.

Reply:
(but also, nice patch for `os.environ`!)